### DISCUSSION TITLE
Transducers導入？

### DISCUSSION THREAD

**Discussion by fosskers** [chaploud/Sci-Lisp#1] [Ideas]
loopsやmap/filterなどの機能をどうしようと話は先日の Meetup で挙がった気がしますが、Transducersの事はご存じでしょうか？

https://clojure.org/reference/transducers
https://github.com/fosskers/cl-transducers/blob/master/README.org

**Comment by chaploud** [comment]
コメント遅れてすみません！
Transducersは知っていましたが、理解が浅いですね。関数型パラダイムとして重要そうなので、盛り込むことを検討します！
